### DISCUSSION TITLE
LibreWolf issue (no more shortcuts in new page)

### DISCUSSION THREAD

**Discussion by guastard0** [xiaoxiaoflood/firefox-scripts#322] [General Support] [answered]
Greetings, kudos for your work. 
Following the installation instructions I managed to get Tab Mix Plus working on LibreWolf. It seems to work fine, the only problem (not related to the extension) is that the browser instead of showing the normal default page (with the shortcuts and the possible "Search the web" field) shows an error message: "something went wrong loading this content".

I tried to clear startup cache on about:support but this doesn't help. If I open the console with F12 it tells me that the problem is in react-dom.js, it says: "TypeError sectionOrder is undefined".
Here my almost non-existent skills end, in the sense that I have no idea what to do to solve the problem.
Any ideas? Thank you in advance

**Comment by xiaoxiaoflood** [comment]
This must be TabMixPlus issue, I suggest you to report it here:

https://github.com/onemen/TabMixPlus/issues

**Reply by guastard0** [reply, marked as answer]
I managed to solve it somehow. I mention this in case someone else runs into the same problem with LibreWolf.
So, I was sure it wasn't TabMixPlus' fault because the error with the page was there BEFORE installing the extension.
I found out that the problem was the general.config.filename string in config-prefs.js: by calling config.js, it replaced the default value of Librewolf which calls its own config file.
So I copied ALL the settings in librewolf.cfg and pasted them in config.js (in addition to and not in place of the original content, of course). Amazingly, it seems to have worked. Now I'm using TabMixPlus in LibreWolf thanks to your awesome scripts :)
